src/impl/select.js

```diff
@@ -1,7 +1,6 @@
 // @flow
-import has from 'lodash/get';
-import range from 'lodash/range';
-import shuffle from 'lodash/shuffle';
+import { has } from '../helpers/object';
+import { range, shuffle } from '../helpers/array';
```

Comment:
I wouldn't care fro jsperf unless we are actually processing a huge stream of data